felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -314,8 +314,8 @@ func newBPFEndpointManager(
 		m.dp = m
 	}
 
-	if config.FeatureDetectOverrides != nil {
```

Comment:
Isn't this a breaking change for anyone who might have set this already in their detection overrides?

Reply:
@tomastigera I think this is still a problem for anyone who upgrades

Reply:
I do not think that this is an issue as that has not been documented anywhere and it is a half baked feature and was pretty much just for ourselves so that we could merge code and not to worry about it to make mess to anyone. The point of this PR is to switch to the gates before we make the ctlb workaround public.